src/components/select/multi-select/multi-select.component.js

```diff
@@ -385,6 +384,11 @@ const MultiSelect = React.forwardRef(
         }
 
         setTextValue("");
+
+        if (isControlled.current && onChange) {
+          onChange(createCustomEvent([...value, newValue]));
```

Comment:
This will allow a user to add the same option more than once, it might be worth adding a test for this scenario as well so it's caught next time

Reply:
Good catch! Thanks :)